webportal/webapp/src/environments/environment.prod.ts

```diff
@@ -3,14 +3,16 @@ export const environment = {
   production: true,
   STREAMING_URL: 'https://stream.securedatacommons.com/guacamole/?authToken=',
   REGION: 'us-east-1',
-  USER_POOL_ID: 'us-east-1_Y5JI7ysvY',
-  CLIENT_ID: '7t034p7j0aklgipi5kk8ll5udd',
+  USER_POOL_ID: 'us-east-1_1zLqP2XrI',
+  CLIENT_ID: 'gt6jl7jeqogmelu5tm7ce1vnb',
   IDENTITY_PROVIDER: 'USDOT-ADFS',
-  APP_DOMAIN: 'dev-sdc-dot-webportal',
+  APP_DOMAIN: 'prod-sdc-dot-webportal',
   REDIRECT_URL: 'https://portal.securedatacommons.com/index.html',
-  API_ENDPOINT: 'https://cyfca70occ.execute-api.us-east-1.amazonaws.com/api/',
-  LOGIN_GOV_COGNITO_APP_CLIENT_ID: '34vj4387sna9jk31gudpt3b78i',
+  API_ENDPOINT: 'https://bzzlrrsxa4.execute-api.us-east-1.amazonaws.com/api/',
+  LOGIN_GOV_COGNITO_APP_CLIENT_ID: '3mlku34f58oeloqqc3hpiko6l3',
   ACCOUNT_LINK_URL: 'https://2l418womp5.execute-api.us-east-1.amazonaws.com/prod',
   LINK_ACCOUNT_PATH: 'prod-link-account',
   ACCOUNT_LINKED_PATH: 'prod-account-linked',
+  RESET_TEMPORARY_PASSWORD_PATH: 'prod-reset-temporary-password',
+  ENVIRONMENT: 'prod',
```

Comment:
Justin, do we need this `ENVIRONMENT` anymore? Its superseded by `ACCOUNT_LINK_URL` right?

Reply:
Yeah. Wouldn't need it for URL generation. It'd only be useful for informational purposes or doing some sorta logic based on environment _(though doing that sorta thing on the front-end probably isn't ideal)_